package/linux-pam/0006-faillock-pipe.patch

```diff
@@ -0,0 +1,129 @@
+--- a/modules/pam_faillock/faillock.h
++++ b/modules/pam_faillock/faillock.h
+@@ -50,6 +50,7 @@
+ #define TALLY_STATUS_VALID     0x1       /* the tally file entry is valid */
```

Comment:
@igorchornobryvtseev can you please add more details - from where did you take this patch?
It looks like this patch is taken (maybe partially) from the new version of the `pam_faillock` repository.

Reply:
we cannot
we discussed it and it has more problems  then benefits

Reply:
it is not upstsream patch

Reply:
existing `faillock `shows only failed login count, e.g. if deny=3:
- on 3d attempt - `faillock `shows 3 failures, actually user is not locked
- on 4th attempt - `faillock `still shows 3 failures, actually user is locked (and `faillock `knows about it)
That's the reason for my modification